assignments/assignment01.rb

```diff
@@ -38,13 +38,17 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
-  # your implementation here
+  a = s.split(" ")
+  b = a.map {|x| x.upcase[0]+x[1..-1].downcase}
```

Comment:
You could just use x.capitalize instead of doing upcase/downcase. The line then becomes:
b = a.map {|x| x.capitalize}